scripts/setup-e2e.sh

```diff
@@ -10,17 +10,33 @@ if [ -n "${TEST_SUITE_ID}" ]; then
   setup_service java 1.8.222
 
   # this chrome install is not used, however it will install linux deps chrome needs (via apt-get)
-  setup_service google-chrome-stable 118.0.5993.70-1
+  # setup_service google-chrome-stable 127.0.6533.88-1
+  setup_service google-chrome-stable 121.0.6167.85-1
   # uses new chrome for testing installation utility (https://developer.chrome.com/blog/chrome-for-testing/)
   # output format: chrome@118.0.5993.70 /path/to/chrome/binary
-  CHROME_INSTALL=$(npx @puppeteer/browsers install chrome@stable)
-  echo $CHROME_INSTALL
-  # extract installed version
-  export CHROMEDRIVER_VERSION=$(echo $CHROME_INSTALL | awk '{print $1}' | cut -d'@' -f 2)
-  echo $CHROMEDRIVER_VERSION
-  # extract binary path
-  export CHROME_BINARY=$(echo $CHROME_INSTALL | awk '{print $2}')
-  echo $CHROME_BINARY
+  # npm i -g @puppeteer/browsers@1.x
+  # @puppeteer/browsers install chrome@stable]
+
+  # OLD_NPM_REGISTRY=$(npm config get registry)
+  # npm config set registry https://registry.npmjs.org
+  # npm config get registry
+
+  # echo "Running npx @puppeteer/browsers"
+  # npx @puppeteer/browsers install chrome@stable --version
+  # npx @puppeteer/browsers install chrome@stable
+  # echo "Running puppeteer install"
+  # CHROME_INSTALL=$(npx @puppeteer/browsers@1.x install chrome@stable)
+  # echo "CHROME_INSTALL: $CHROME_INSTALL"
+  # # extract installed version
+  # export CHROMEDRIVER_VERSION=$(echo $CHROME_INSTALL | awk '{print $1}' | cut -d'@' -f 2)
+  # echo $CHROMEDRIVER_VERSION
+  # # extract binary path
+  # export CHROME_BINARY=$(echo $CHROME_INSTALL | awk '{print $2}')
+  # echo $CHROME_BINARY
+
+  # npm config set registry $OLD_NPM_REGISTRY
+  # wget -v https://storage.googleapis.com/chrome-for-testing-public/127.0.6533.88/linux64/chrome-linux64.zip
+  # echo $(ls)
```

Comment:
Do we still need these commented lines?

Reply:
theoretically no, but this was pulled in from master
https://github.com/okta/okta-auth-js/blob/master/scripts/setup-e2e.sh#L15